fluent-bit: update template NOTES

From: Arturo Borrero Gonzalez <arturo.bg@arturo.bg>

Update template NOTES.txt file.

Signed-off-by: Arturo Borrero Gonzalez <arturo.borrero@chainguard.dev>
---
 bitnami/fluent-bit/templates/NOTES.txt |    5 -----
 1 file changed, 5 deletions(-)

diff --git a/bitnami/fluent-bit/templates/NOTES.txt b/bitnami/fluent-bit/templates/NOTES.txt
index c75b564..ef13bd5 100644
--- a/bitnami/fluent-bit/templates/NOTES.txt
+++ b/bitnami/fluent-bit/templates/NOTES.txt
@@ -23,8 +23,3 @@ APP VERSION: {{ .Chart.AppVersion }}
     echo "HTTP port at 127.0.0.1:2020"
     kubectl port-forward svc/{{ include "common.names.fullname" . }} 2020:{{ .Values.service.ports.http }} &
 {{- end }}
-
-{{- include "common.warnings.rollingTag" .Values.image }}
-{{- include "common.warnings.resources" (dict "sections" (list "") "context" $) }}
-{{- include "common.warnings.modifiedImages" (dict "images" (list .Values.image) "context" $) }}
-{{- include "common.errors.insecureImages" (dict "images" (list .Values.image) "context" $) }}
